showup/views.py

```diff
@@ -67,37 +67,38 @@ def edit_profile(request, id):
         raise PermissionDenied
 
 
+def get_stack(request, eid):
+    # My uid.
+    uid = request.user.id
+
+    # Get the users interested in or going to the event.
+    users = (CustomUser.objects.filter(interested__id=eid) |
+             CustomUser.objects.filter(going__id=eid))
+
+    # These are the users that swiped left on me.
+    swiped_left = [x.swiper.id for x in Swipe.objects.filter(
```

Comment:
please don't change variable names. unless you're removing the code entirely. This affects all current branches and PRs

Reply:
Not sure what you mean. I pulled this code and made an entirely different function (`get_stack()`) so changing the variable names won't cause any errors.

Reply:
my bad. git highlighted looked like that variable have been renamed.